jms/1.7/modules/ROOT/pages/jms-topic-subscription.adoc

```diff
@@ -1,17 +1,29 @@
-= Using Topic Subscriptions
+= Configure Topic Subscriptions with JMS Connector
 :keywords: jms, connector, consume, source, listener, topic, subscription
 :page-aliases: connectors::jms/jms-topic-subscription.adoc
 
+Topics in JMS enables you to configure how the subscription behaves regarding the connection of the consumer and also how these consumers interact with each other.
 
+The Anypoint Connector for JMS (JMS Connector) *Consumer* and *On New Message* operations can manage the aspects of a topic subscription in the *Consumer type* field by selecting the *Topic consumer* configuration.
 
-Topics in JMS allow you to configure how the subscription behaves regarding the connection of the consumer and also how this consumers interact between each other.
+== Configure Durable Subscriptions
 
-Using Mule's JMS Connector you can handle of the aspects of a Topic subscription using the 'topic-consumer' configuration in both the `consumer` and `listener` operations.
+Durable subscriptions enables you to receive published messages to the topic while the consumer is offline.
+Once you configure a durable subscription, an available consumer receives all the published messages before the established connection. +
+Then the JMS provider stores the messages sent or published to the topic, as it would store messages sent to a queue. If another application has a topic consumer configuration that uses the same connection and its client ID, the same topic, and the same subscription name, the subscription reactivates, and the JMS provider delivers the published messages while the subscriber was inactive.
```

Comment:
To what does "Then" refer? What comes before this to make this later?

Reply:
Best guess:
```suggestion
The JMS provider stores messages sent or published to a topic, just as it would messages sent to a queue. If another consumer uses the same connection, client ID, topic, and subscription name, the JMS provider delivers the published messages, even if while the subscriber is inactive.
```
But I sort of want an illustration here.
Are we using "consumer", "topic consumer configuration," and "subscriber" to mean the same thing here: the recipient of the message? Are the messages "sent" or "published"? Supplying alternative terms makes it difficult, at least for me, to follow the narrative.